ql/src/test/queries/clientpositive/udf_configurable.q

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE test_udf_configurable (cint1 INT, cint2 INT, ctimestamp TIMESTAMP, text_timestamp STRING);
+INSERT INTO test_udf_configurable VALUES
+  (10000, 3, CAST('1970-01-01 01:02:03' AS TIMESTAMP), '1970-01-01 01:02:03 4'),
+  (20000, 5, CAST('1970-01-02 04:05:06' AS TIMESTAMP), '1970-01-02 04:05:06 5'),
+  (30000, 7, CAST('1970-01-03 07:08:09' AS TIMESTAMP), '1970-01-03 07:08:09 6');
+
+set hive.compat=latest;
+set hive.strict.timestamp.conversion=false;
+set hive.int.timestamp.conversion.in.seconds=true;
+set hive.local.time.zone=Asia/Bangkok;
+set hive.datetime.formatter=SIMPLE;
+set hive.masking.algo=sha512;
+set hive.use.googleregex.engine=true;
+
+-- On HiveServer2
+SELECT
+  -- DECIMAL because of hive.compat=latest
+  TYPEOF(cint1 / cint2),
+  -- Allowed by hive.strict.timestamp.conversion=false
+  cint1 = ctimestamp,
```

Comment:
nit: the left side of `cint1 = ctimestamp` shouldn't be converted to timestamp,  right?